.gitignore

```diff
@@ -23,5 +23,8 @@ osd_scenarios_*node*
 version_info.json
 rerun
 sut.yaml
+logs
+osp
```

Comment:
Thanks for breaking them into individual commits. It would be helpful if there can be more description in them though.

Could you let me know the modules generating these residues?